experiments/ClimaEarth/components/land/climaland_helpers.jl

```diff
@@ -0,0 +1,59 @@
+import ClimaCore as CC
+import ClimaLand
+"""
+    temp_anomaly_aquaplanet(coord)
+
+Introduce a temperature IC anomaly for the aquaplanet case.
+The values for this case follow the moist Held-Suarez setup of Thatcher &
+Jablonowski (2016, eq. 6), consistent with ClimaAtmos aquaplanet.
+"""
+temp_anomaly_aquaplanet(coord) = 29 * exp(-coord.lat^2 / (2 * 26^2))
+
+"""
+    temp_anomaly_amip(coord)
+
+Introduce a temperature IC anomaly for the AMIP case.
+The values used in this case have been tuned to align with observed temperature
+and result in stable simulations.
+"""
+temp_anomaly_amip(coord) = 40 * cosd(coord.lat)^4
+
+"""
+    make_land_domain(
+        atmos_boundary_space::CC.Spaces.SpectralElementSpace2D,
+        zlim::Tuple{FT, FT},
+        nelements_vert::Int,) where {FT}
+
+Creates the land model domain from the horizontal space of the atmosphere, and information
+about the number of elements and extent of the vertical domain.
+"""
+function make_land_domain(
+    atmos_boundary_space::CC.Spaces.SpectralElementSpace2D,
```

Comment:
A note that when ClimaLand uses a mask in the space, the atmos still will not. On our side, we will need to both set enable_mask = true and update the atmos space's mask in place with the correct values. The latter we can do with an existing space, but I don't think we can enable the mask once we've made a space without it.

so this function will get more complicated

Reply:
Another note to make is that ClimaLand (when running in standalone) will always use GL quadrature and npolynomial = 0. this will be different than Atmos. We may be making some assumptions in ClimaCore mask capabilities and interpolation that this is the case. 

That said, to start with, we can run coupled simulations without ClimaLand using a mask and using whatever space the atmos is using. ClimaLand will be much slower in that case, but it should be okay